src/components/molecules/VNavigation.vue

```diff
@@ -0,0 +1,89 @@
+<template>
+  <nav class="nav">
+    <ul class="nav__ul">
+      <li>
```

Comment:
please make an array containing menus inside the `computed` property so you'll be able to loop them 

```
...
computed() {
  menus: [
    {
      urlPath: '/',
      iconComponent: 'v-smart-home-icon'
    },
    {
      urlPath: '',
      iconComponent: 'v-calendar-time-icon'
    },
    {
      urlPath: '',
      iconComponent: 'v-chart-area-line-icon'
    },
  ]
}
...
```

```
<template>
  ...
  <li v-for="menu in menus" class="nav__li">
    <router-link :to="menu.urlLink" exact>
      <component :is="menu.iconComponent" class="nav__icon" />
      <span :class="{ 'nav__li--is-active': $route.path === menu.urlLink }" />
    </router-link>
   </li>
  ...
</template>
```

Reply:
when you see something redundant, you can always create an array and loop them

Reply:
this is great!! I didn't think about that before. thank you very much 
next time I'm going to be more careful to use code efficiently